pkgs/top-level/all-packages.nix

```diff
@@ -11750,6 +11752,16 @@ in
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  libgit2_0_27 = libgit2.overrideAttrs (oldAttrs: rec {
+    version = "0.27.8";
```

Comment:
This version is insecure. You need to update to https://github.com/libgit2/libgit2/releases/tag/v0.27.10, or better https://github.com/libgit2/libgit2/releases/tag/v1.0.0.

Reply:
Or add

```
meta = oldAttrs.meta // {
  knownVulnerabilities = [
    "CVE-2019-1348"
    "CVE-2019-1349"
    "CVE-2019-1350"
    "CVE-2019-1351"
    "CVE-2019-1352"
    "CVE-2019-1353"
    "CVE-2019-1354"
    "CVE-2019-1387"
  ];
};
```

Reply:
I switched it to 0.27.10

Reply:
Please also open an upstream issue since it is  not clear how long it will remain supported.

Reply:
I created this issue : https://github.com/isacikgoz/gitin/issues/63